src/Transports.AspNetCore/IUserContextBuilder.cs

```diff
@@ -1,16 +1,17 @@
 using Microsoft.AspNetCore.Http;
 
-namespace GraphQL.Server.Transports.AspNetCore;
-
-/// <summary>
-/// Interface which is responsible of building a UserContext for a GraphQL request
-/// </summary>
-public interface IUserContextBuilder
+namespace GraphQL.Server.Transports.AspNetCore
 {
     /// <summary>
-    /// Builds the UserContext using the specified <see cref="HttpContext"/>
+    /// Interface which is responsible of building a UserContext for a GraphQL request
```

Comment:
```suggestion
    /// Interface which is responsible of building a UserContext for a GraphQL request.
```

Reply:
This entire file had not been updated to my copy.  I have done so; see updated comments.